src/uci.cpp

```
@@ -0,0 +1,437 @@
+    return this->defaultValue;
+}
+
+void UCIOption::setDefaultValue(const std::string& value) {
+    this->defaultValue = value;
+}
+
+std::string UCIOption::getMinValue() {
+    return this->minValue;
+}
+
+void UCIOption::setMinValue(std::string value) {
```

cppcheck

[performance] Function parameter 'value' should be passed by const reference.

(passedByValue)